ShowerShow/Model/Preferences.cs

```diff
@@ -1,14 +1,37 @@
 ﻿using Newtonsoft.Json;
 using System;
+using System.Diagnostics;
 
 namespace ShowerShow.Models
 {
-    public class Preferences
+    public partial class Preferences
     {
         public Guid Id { get; set; } = Guid.NewGuid();
         [JsonRequired]
         public Guid UserId { get; set; }
         public AvailableVoices SelectedVoice { get; set; } = AvailableVoices.Default_Male;
         public AvailableLanguages SelectedLanguage { get; set; } = AvailableLanguages.English;
```

Comment:
Voices are in blob storage. Check in with cosmin to understand it better. this class needs a tiny update